### DISCUSSION TITLE
Support for evidence report from Cdxgen

### DISCUSSION THREAD

**Discussion by phoenix-aditya** [DependencyTrack/dependency-track#3005] [General]
Hey,

cdxgen introduced a feature for providing evidence for the dependency report it generates, I wanted to know that will dependency track provide a visual interface for the evidences and services report cdxgen provides and what would the timeline be for the same?

**Comment by pinkfloydx33** [comment]
Is there any value (as far as Dependency-track is concerned) for us to use the evidence argument to cdxgen? If DT won't do anything with it, then it's probably best to ignore since it takes considerable time to run... But unsure if there's value or if DT will/can use
